docs/snippets/react/list-story-expanded.js.mdx

```diff
@@ -8,22 +8,27 @@ import ListItem from './ListItem';
 
```

Comment:
for the above lines 😄 
```
import { List } from './List';
import { ListItem } from './ListItem':
```

Reply:
I'll leave it as it is if you don't mind, as per the comment above. As I wrote it as a default export and I cannot know if the reader uses named exports per his/her development patterns.